src/main/kotlin/com/sourcegraph/cody/autocomplete/CodyAutocompleteManager.kt

```diff
@@ -256,13 +255,13 @@ class CodyAutocompleteManager {
               null
             }
             .completeOnTimeout(null, 3, TimeUnit.SECONDS)
-            .get()
-
-        resetApplication(project)
-        resultOuter.complete(null)
+            .thenRun {
```

Comment:
lgtm, but pls double check that autocomplete works fine

Reply:
I checked!